db/db_test.cc

```diff
@@ -646,7 +646,7 @@ TEST_F(DBTest, SingleDeleteFlush) {
 
     ASSERT_EQ("NOT_FOUND", Get(1, "bar"));
     ASSERT_EQ("NOT_FOUND", Get(1, "foo"));
-    // Skip FIFO and universal compaction beccause they do not apply to the test
+    // Skip FIFO and universal compaction beccaus they do not apply to the test
```

Comment:
Still appears to be incorrect? :)

Reply:
lol, good catch, fixed my typo :)

Reply:
Did you forget to update the PR?

Reply:
@siying oh, my second commit fixed the typo.